packages/combo-box/test/lazy-loading.test.js

```diff
@@ -1029,7 +1029,7 @@ describe('lazy loading', () => {
         let returnedItems;
 
         const bluringDataProvider = (params, callback) => {
-          comboBox.blur();
+          comboBox.inputElement.blur();
```

Comment:
**note:** The combo-box-light doesn't delegate `blur()` calls to the input element.